api/src/main/java/io/grpc/MethodDescriptor.java

```diff
@@ -265,7 +265,7 @@ public String getServiceName() {
   /**
    * A convenience method for {@code extractBareMethodName(getFullMethodName())}.
    *
-   * @since 1.32.0
+   * @since 1.33.0
```

Comment:
How about making the same fix on line 416 for `extractBareMethodName` ?

Reply:
@sanjaypujare Thank you for your feedback. Indeed, I did not look at the `extractBareMethodName`... I made the requested change and also updated the description of the PR.